website/versions.json

```diff
@@ -1 +1 @@
-["v2.0_alpha", "v1.2", "v0.x"]
+["v2.x", "v1.2", "v0.x"]
```

Comment:
not sure if it's just me but I cannot see the left navigation bar for the docs on v2.x

Reply:
nice, it shows now for me too, not sure what happened there haha